api/v1beta1/hyperconverged_types.go

```diff
@@ -169,6 +169,10 @@ type HyperConvergedSpec struct {
 	// +optional
 	TektonPipelinesNamespace *string `json:"tektonPipelinesNamespace,omitempty"`
 
+	// TektonTasksNamespace defines namespace in which tekton tasks will be deployed.
+	// +optional
+	TektonTasksNamespace *string `json:"tektonTasksNamespace,omitempty"`
```

Comment:
do we need a default value here?

Reply:
Currently in `tto.go` [1] there is a check if that nil, it'll set the default value of `hc.Namespace` by using `getNamespace` function:
```
func NewTTO(hc *hcov1beta1.HyperConverged, opts ...string) *ttov1alpha1.TektonTasks {
	pipelinesNamespace := getNamespace(hc.Namespace, opts)
	if hc.Spec.TektonPipelinesNamespace != nil {
		pipelinesNamespace = *hc.Spec.TektonPipelinesNamespace
	}
...
}
```
Should we do the same thing here?

[1] https://github.com/kubevirt/hyperconverged-cluster-operator/blob/main/controllers/operands/tto.go#L83-L86

Reply:
Done, I've added that code below to `ssp.go`:
```
    // Default value is the operator namespace
    pipelinesNamespace := getNamespace(hc.Namespace, opts)
    if hc.Spec.TektonPipelinesNamespace != nil {
        pipelinesNamespace = *hc.Spec.TektonPipelinesNamespace
    }

    spec.TektonPipelines.Namespace = pipelinesNamespace

    // Default value is the operator namespace
    tasksNamespace := getNamespace(hc.Namespace, opts)
    if hc.Spec.TektonTasksNamespace != nil {
        tasksNamespace = *hc.Spec.TektonTasksNamespace
    }

    spec.TektonTasks.Namespace = tasksNamespace
```